tests/dist/python/run_dist_objects.py

```diff
@@ -695,12 +695,20 @@ def test_dist_dataloader(g):
         "DistDataLoader": test_dist_dataloader,
     }
 
-    target = os.environ.get("DIST_DGL_TEST_OBJECT_TYPE", "")
-    if target not in target_func_map:
+    targets = os.environ.get("DIST_DGL_TEST_OBJECT_TYPE", "")
+    targets = targets.replace(' ', '').split(',') if targets else []
+    blacklist = os.environ.get("DIST_DGL_TEST_OBJECT_TYPE_BLACKLIST", "")
+    blacklist = blacklist.replace(' ', '').split(',') if blacklist else []
+
+    for to_bl in blacklist:
+        target_func_map.pop(to_bl, None)
+
+    if not targets:
         for test_func in target_func_map.values():
             test_func(g)
     else:
-        target_func_map[target](g)
+        for target in targets:
+            target_func_map[target](g)
```

Comment:
if an unknown test is specified, what does the error message look like? could we print a readable message?

Reply:
Done